whois-api/src/main/java/net/ripe/db/whois/api/rdap/RedactionObjectMapper.java

```diff
@@ -0,0 +1,58 @@
+package net.ripe.db.whois.api.rdap;
+
+import net.ripe.db.whois.api.rdap.domain.Redaction;
+import net.ripe.db.whois.api.rdap.domain.Role;
+import net.ripe.db.whois.common.rpsl.AttributeType;
+import net.ripe.db.whois.common.rpsl.RpslAttribute;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+import org.springframework.stereotype.Component;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import static net.ripe.db.whois.common.rpsl.AttributeType.NOTIFY;
+import static net.ripe.db.whois.common.rpsl.AttributeType.MBRS_BY_REF;
+import static net.ripe.db.whois.common.rpsl.AttributeType.MNT_DOMAINS;
+import static net.ripe.db.whois.common.rpsl.AttributeType.MNT_LOWER;
+import static net.ripe.db.whois.common.rpsl.AttributeType.MNT_REF;
+import static net.ripe.db.whois.common.rpsl.AttributeType.MNT_ROUTES;
+
+@Component
+public class RedactionObjectMapper {
+
+    private static final Map<AttributeType, AttributeTypeRedaction> UNSUPPORTED_REGISTRANT_ATTRIBUTES = Map.of(
+            MBRS_BY_REF, AttributeTypeRedaction.MBRS_BY_REF,
+            MNT_DOMAINS, AttributeTypeRedaction.MNT_DOMAINS,
+            MNT_LOWER, AttributeTypeRedaction.MNT_LOWER,
```

Comment:
I still feel creating an enum is overkill and its also confusing when attributeType and AttributeTypeRedaction have same value. If we had to send this enum in a json response It would have make more sense to me. But feel free to ignore

Reply:
¬¬ 
done, for me it is the same. Even switch will be loaded in memory as the enum will because both are static. 
But if you feel more comfortable with the switch it is fine